step.yml

```diff
@@ -9,6 +9,11 @@ source_code_url: https://github.com/bitrise-steplib/steps-virtual-device-testing
 support_url: https://github.com/bitrise-steplib/steps-virtual-device-testing-for-ios/issues
 project_type_tags:
   - ios
+  - cordova
+  - ionic
+  - react-native
+  - flutter
+  
```

Comment:
xamarin is possible?

Reply:
No, it's not. For that, we should use Xcovebuild build for testing which is not working for Xamarin projects.